brave/browser-laptop
# Adds captcha block overlay

Resolves #14356

## Submitter Checklist:


## Test Plan:
1. enable payments
2. claim captcha
3. try to solve captcha more then 30 times
4. you should get too many attempts error

![image](https://user-images.githubusercontent.com/9574457/41141669-38036e18-6af3-11e8-9b04-9e3e88d0ccff.png)

## Reviewer Checklist:


Tests
